loop/conftest.py

```diff
@@ -1,15 +1,79 @@
-# Configuration file for running our tests
+import ConfigParser
 import pytest
 
+from firefox_puppeteer import Puppeteer
+from marionette_driver.marionette import Marionette
+TIMEOUT = 20
+
 
 def pytest_addoption(parser):
+    """Options to be passed in at run time"""
     parser.addoption(
         "--env",
         action="store",
         help="choose a test environment: staging or production"
```

Comment:
We should try and standardize the names for our test environments:

- dev
- stage
- prod